Implementacija/savetovaliste/app/Controllers/Korisnik.php

```diff
@@ -2,11 +2,130 @@
 
 namespace App\Controllers;
 
+use App\Models\PitanjeModel;
+use App\Models\OdgovorModel;
+use App\Models\KorisnikModel;
+use App\Models\KategorijaPitanjaModel;
+
 class Korisnik extends BaseController
 {
 	public function index()
 	{
 		echo view("pocetna_stranica");
 		//return view('welcome_message');
 	}
+
+	// zelim da odgovorim na pitanje ciji je id proslednjen
+	// prikazace se forma za cuvanje novog odgovora na odabrano pitanje
+	public function odgovori_na_pitanje($idPitanje){
+		$pitanjeModel = new PitanjeModel();
+		$pitanje = $pitanjeModel->find($idPitanje);
+		echo view("odgovori_na_pitanje", ['pitanje' => $pitanje]);
+	}
+
+	// kada se submit forma poziva se ova funkcija kako bi se sacuvao odgovor u bazi
+	public function odgovoriNaPitanje($idPitanje){
```

Comment:
Odlazak na link `localhost:8080/Korisnik/odgovoriNaPitanje/` izaziva crash. Ja bih ovde inicijalizovao $idPitanje = null, pa posle proverio ako je u kodu null, redirectovao bih...
```suggestion
	public function odgovoriNaPitanje($idPitanje = null){
	if($idPitanje == null)
	    // return redirect na stranicu sa odgovorima
```

Reply:
I nepostojeći id isto crashuje.
`http://localhost:8080/Korisnik/odgovoriNaPitanje/555`

Reply:
Sredjena oba